CMakeLists.txt

```diff
@@ -1,11 +1,17 @@
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 3.10)
 project(stage_ros)
 
-find_package(catkin REQUIRED
-  COMPONENTS
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_CXX_EXTENSIONS OFF)
+
+find_package(Boost REQUIRED COMPONENTS system thread)
+find_package(stage REQUIRED)
```

Comment:
Is it possible to put STAGE in the dependencies list?

Reply:
It is, but only by putting it in full capitals. Since it's not a ament / colcon package anyway, I prefer to keep it like this.